encryption_works.md

```diff
@@ -98,25 +98,41 @@ Tails, which is a live DVD and live USB GNU/Linux distribution that I will discu
 
 ## Anonymize Your Location with Tor
 
-Tor is a software service that allows you to use the Internet while concealing your IP address, which is, in general, a fairly accurate representation of your location. The [Tor](https://www.torproject.org/) network is made up of over 3,600 volunteer servers called nodes. When someone uses the Tor network to visit a website their connection gets bounced through three of these nodes (called a circuit) before finally exiting into the normal Internet. Anyone intercepting traffic will think your location is the final node which your traffic exits from.
+Despite NSA surveillance, it’s still possible to browse the web with a high degree of anonymity. The Tor Browser is a web browser like Firefox or Safari, but it allows you to use the Internet while concealing your IP address, which is generally an accurate representation of your approximate location. 
```

Comment:
I would avoid making an absolute statements like "high degree" or that by using Tor you can evade NSA surveillance (they still get plenty of Tor users via targeted surveillance). Also remember the NSA docs were from years ago now and it may not have even been all the NSA knew about Tor at the time. (You'll notice Micah purposefully makes a lot of hedging statements throughout the document, we should keep those in there, even if we simplify them if they are too confusing.) My suggested change to this paragraph:

"While your web habits and location can often be tracked when browsing the Internet, it’s still possible to browse the web anonymously and avoid many types of mass surveillance. The easiest way is with the Tor Browser, a web browser like Firefox or Safari, but it allows you to use the Internet while masking your IP address, which is generally an accurate representation of your approximate location."